File: BEE 1035.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);

        int a = input.nextInt();
        int b = input.nextInt();
        int c = input.nextInt();
        int d = input.nextInt();

        if ((b > c) && (d > a) && (c+d > a+b) && (c >= 0 && d >= 0) && (a%2 == 0)){
            System.out.printf("Valores aceitos\n");
        }
        else{
            System.out.printf("Valores nao aceitos\n");
        }
    }
}

File: BEE 1035.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var a = parseInt(line.shift());
var b = parseInt(line.shift());
var c = parseInt(line.shift());
var d = parseInt(line.shift());

if ((b > c) && (d > a) && (c+d > a+b) && (c >= 0 && d >= 0) && (a%2 == 0)){
    console.log("Valores aceitos");
}
else{
    console.log("Valores nao aceitos");
}